src/con_duct/suite/plot.py

```diff
@@ -23,7 +24,13 @@ def matplotlib_plot(args: argparse.Namespace) -> int:
         try:
             with open(file_path, "r") as info_file:
                 info_data = json.load(info_file)
-                file_path = info_data["output_paths"]["usage"]
+                rel_usage_path = info_data["output_paths"]["usage"]
+
+                # determine usage abspath
+                rel_info_path = info_data["output_paths"]["info"]
+                abs_info_path = os.path.abspath(file_path)
+                original_wd = abs_info_path[: -len(rel_info_path)].rstrip(os.sep)
+                file_path = os.path.join(original_wd, rel_usage_path)
```

Comment:
managing "paths" manually with `[:]` string operations fragile, and better be avoided. I think the best would be to start using more libpath here... under assumption of them being in the same folder (we have it a "base assumption" so I think it is ok to just add `assert` if really paranoid), could even be as simple as a single `.with_name`.

```
In [24]: p1 = Path('/a/b/c/d/e/f'); rp1 = Path('e/f'); rp2 = Path('e/g')

In [25]: assert rp1.parent == rp2.parent

In [26]: p1.with_name(rp2.name)
Out[26]: PosixPath('/a/b/c/d/e/g')
```